packages/api/src/command/connected-user/get-connected-user.ts

```diff
@@ -77,3 +77,26 @@ export const getConnectedUserByTokenOrFail = async (
 
   return connectedUser;
 };
+
+export const getConnectedUsersByTokenOrFail = async (
+  provider: string,
+  token: string
+): Promise<ConnectedUser[]> => {
+  const connectedUsers = await ConnectedUser.findAll({
+    where: {
+      providerMap: {
+        [provider]: {
+          token: {
+            // TODO: Find more optimal solution
+            [Op.like]: "%" + token + "%",
+          },
+        },
+      },
+    },
+  });
+
+  if (!connectedUsers)
+    throw new NotFoundError(`Could not find connected users with str matching token`);
```

Comment:
This seem to be based off the other, similar function. Do we want to fail when it returns falsy (null/undefined), or when it doesn't find any records? AFAIK, the former is not possible (it always return an array), no?

Also, the message refers to `str` which is not the name of the param of this function.

Reply:
Right. We gotta check for the length of connectedUsers. Good catch :)